.buildkite/pipeline.yml

```
@@ -1,3 +1,14 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
+
+env:
+  SETUP_MAGE_VERSION: '1.14.0'
+
 steps:
-  - label: "Example test"
-    command: echo "Hello!"
+  - label: "Checks formatting / linting"
+    commands:
+      - ".buildkite/scripts/install_dependencies.sh"
+      - "mage -debug check"
```

Not sure if they are executed in the same shell. Maybe the PATH environment set in the `install_dependencies.sh` is not present in the second command.

You could test to include in the same line those two commands:

```suggestion
      - ".buildkite/scripts/install_dependencies.sh && mage -debug check"
```